docs/quickstart.rst

```diff
@@ -71,11 +71,14 @@ can be installed easily via `pip` using a pip extras option.
 Here's a list of the components and their extras option:
 
 * Amazon Web Services (Cloud) : `aws`
+* OAuth based SSH : `oauth_ssh`
+* Extreme Scale Executor (Supercomputing) : `extreme_scale`
 * Google Cloud : `google_cloud`
+* Python GSSAPI for SSH : `gssapi`
+* Azure (Cloud) : `azure`
 * Kubernetes : `kubernetes`
-* Extreme Scale Executor (Supercomputing) : `extreme_scale`
-* Logging monitoring data to a database: `monitoring`
-* Jetstream (NSF Cloud) : `jetstream`
+* Work Queue execution framework : `workqueue`
+
```

Comment:
There is also a target for `gssapi`, I guess to do with the ssh channel?

Reply:
Yes, that's right.